### ISSUE TITLE
Support go to the first and last group keyboard shortcut

### ISSUE THREAD

**Issue by jooyoungseo** [julianna-langston/chart2music#51]
Page up/down keys move between groups; however, I have not been able to find a way to go to the first group and the last group quickly, such as `Ctrl+PG-Up/Down`. It would be great if these shortcuts could be added.

**Comment by julianna-ciq** [comment]
I have implemented the ability to jump to the first and last category/group. However, I wasn't able to use Ctrl+PageUp and Ctrl+PageDown. Those keyboard shortcuts  are already used by browsers to move between browser tabs. Instead, I went with Alt+PageUp and Alt+PageDown. (On a Mac, the "Alt" would be "Option").

This functionality will be part of version 1.5, once we release it. That will probably be this weekend.
